client/webpack.config.js

```diff
@@ -114,16 +114,6 @@ module.exports = function exports(env, argv) {
     'panels',
     'streamfield.scss',
   );
-  sassEntry[getOutputPath('contrib/styleguide', 'css', 'styleguide')] =
-    path.resolve(
-      'wagtail',
-      'contrib',
-      'styleguide',
-      'static_src',
-      'wagtailstyleguide',
-      'scss',
-      'styleguide.scss',
```

Comment:
We need to remove this line as well:

https://github.com/wagtail/wagtail/blob/ca83e5a34f960ba53ccded45a59884eee054af60/wagtail/contrib/styleguide/templates/wagtailstyleguide/base.html#L7

Reply:
Thank you!